maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -103,94 +93,11 @@ private void testing(File runFile) throws MojoExecutionException, Exception, IOE
 			}
 
 			if (resolve) {
-				resolve(run);
+				run.resolve(failOnChanges);
```

Comment:
> Would this mean that the new resolve plugin operation is not tied, by default, to any maven lifecycle and would need to be invoked manually?

Yes. Just like in Gradle and Bndtools. Running the resolve task is an explicit operation since it mutates a source file.